storage_v2/factoryadapter/reader.go

```diff
@@ -37,8 +37,14 @@ func NewTraceReader(spanReader spanstore.Reader) *TraceReader {
 	}
 }
 
-func (*TraceReader) GetTrace(_ context.Context, _ pcommon.TraceID) (ptrace.Traces, error) {
-	panic("not implemented")
+func (tr *TraceReader) GetTrace(ctx context.Context, traceID pcommon.TraceID) (ptrace.Traces, error) {
+	tid, _ := model.TraceIDFromBytes(traceID[:])
```

Comment:
@yurishkuro are we okay to ignore this error? or do we want to handle it? An error is only returned if the length of the traceID isn't 8 or 16 and `pcommon.TraceID` is defined as `type TraceID [16]byte`

Reply:
If it's easy to test I wouldn't ignore it.

Reply:
@yurishkuro Its not possible to because the length of traceID is always 16 so the error will never be triggered. Let me know if I'm missing something though.

Reply:
if len=16 is guaranteed, I would just create TraceIDFromOTEL function that doesn't return an error.

Reply:
@yurishkuro thanks for the suggestion - done